emails/views.py

```diff
@@ -513,15 +513,19 @@ def _sns_message(message_json):
     message_body = {}
     tracker_report_link = ""
     removed_count = 0
+    datetime_now = int(datetime.now(timezone.utc).timestamp() * 1000)
```

Comment:
I think it should have the comment from below (lines 530-532):

```suggestion
    # frontend expects a timestamp in milliseconds
    datetime_now = int(datetime.now(timezone.utc).timestamp() * 1000)
```

I also think lines 530-532 should be deleted, so that the same timestamp is used in the tracker report link in the footer.

As I said before, I like setting `datetime_now` once and using it for all the tracker links. 👍